test/functionalTest/cases/3000_allow_creation_transient_entities/create_transient_entity.test

```diff
@@ -0,0 +1,115 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Create a new transient entity, by specifying dateExpires attribute.
+Sleep 60 seconds, the default time interval for Mongo TTL monitor to remove expired entity.
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. POST /v2/entities with dateExpires attribute (expired ISO8601 date value)
+# 02. GET /v2/entities the entity is present (expired but not removed yet)
+# 03. GET /v2/entities after 60 seconds (default removal interval of mongo TTL thread)
+# and check if the entity was removed.
+#
+
+echo "01. POST /v2/entities with dateExpires attribute (expired ISO8601 date value)"
+echo "====================================================================="
```

Comment:
Style: please align the number of "=" in the second echo with the length of the string in the first echo.

Reply:
(General comment in all .test, it seems it is happening in other places)

Reply:
Fixed in faea551